br/pkg/lightning/web/progress.go

```diff
@@ -93,10 +93,16 @@ const (
 )
 
 type tableInfo struct {
-	TotalWritten int64      `json:"w"`
-	TotalSize    int64      `json:"z"`
-	Status       taskStatus `json:"s"`
-	Message      string     `json:"m,omitempty"`
+	TotalWritten int64           `json:"w"`
+	TotalSize    int64           `json:"z"`
+	Status       taskStatus      `json:"s"`
+	Message      string          `json:"m,omitempty"`
+	Progresses   []tableProgress `json:"progresses,omitempty"`
```

Comment:
Currently, only the `add-index` progress will be reported, right ?

Reply:
Yes.